src/test/java/core/basesyntax/strategy/impl/BalanceOperationCalculatorImplTest.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.strategy.OperationCalculator;
+import org.junit.Test;
+
+public class BalanceOperationCalculatorImplTest {
+    private static final int VALID_BALANCE = 28;
+    private static final int VALID_QUANTITY = 13;
+    private OperationCalculator balanceOperationCalculator;
+
+    @Test
```

Comment:
add negative tests

Reply:
Balance Operation has two input values and return one of them. What negative tests are you talking about? I didn't come up with any.

Reply:
null value balance,  null value quantity, null value both values, 0 values